include/interface/drivers/SPI.h

```diff
@@ -0,0 +1,24 @@
+// Leka - LekaOS
+// Copyright 2021 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _LEKA_OS_INTERFACE_DRIVER_SPI_H_
+#define _LEKA_OS_INTERFACE_DRIVER_SPI_H_
+
+#include <stdint.h>
```

Comment:
c'est mieux d'utiliser les headers C++

```suggestion
#include <cstdint>
```